product_docs/docs/pgd/4/rel_notes/pgd_4.3.0_rel_notes.mdx

```diff
@@ -10,6 +10,7 @@ EDB Postgres Distributed version 4.3.0 is a patch release of EDB Postgres Distri
 
 | Component | Version |  Type           |                                                   Description                                                           |
 | --------- | ------- | --------------- | ------------------------------------------------------------------------------------------------------------------------|
+| BDR       | 4.3.0   | Feature         | Add pid to the log message emitted upon writer process death (BDR-2534) |
```

Comment:
Huh? why is this both in 4.3.0 and 4.3.1?

Reply:
no, this is only 4.3.1
Removed